src/Shell/CacheShell.php

```diff
@@ -1,40 +0,0 @@
-<?php
-/**
- * BEdita, API-first content management framework
- * Copyright 2018 ChannelWeb Srl, Chialab Srl
- *
- * This file is part of BEdita: you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as published
- * by the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
- */
-
-namespace App\Shell;
-
-use Cake\Filesystem\Folder;
-use Cake\Shell\CacheShell as BaseCacheShell;
-
-/**
- * Extend `CacheShell::clearAll` to remove Twig compiled files.
- */
-class CacheShell extends BaseCacheShell
```

Comment:
`Twig` compiled templates are not cleared with `bin/cake cache clearAll`
what about an empty `CacheShell` extending `BEdita\Webtools` one?